scripts/java-maven-gradle-sdkman.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+apt-get update
+apt purge default-jdk java-common
```

Comment:
```suggestion
apt-get purge default-jdk java-common
```

Histoire d'être uniforme et `apt-get` est plus pour le scripting tandis que `apt` plus pour le terminal interactif